test/unit/package_set_test.rb

```diff
@@ -33,38 +41,38 @@ class PackageSetTest < Minitest::Test
     end
 
     test ".package_paths supports a path wildcard" do
-      package_paths = PackageSet.package_paths("test/fixtures/skeleton", "**")
+      package_paths = PackageSet.package_paths(".", "**")
```

Comment:
While `use_template(:x)` is in effect, the current working directory changes to `tmp/fixtures/the_unique_fixture_copy_for_this_test`, which is the root of the application.